features/ownership/suggestion.feature

```diff
@@ -0,0 +1,50 @@
+Feature: Test ownership suggestions
+  Background:
+    Given I am using the UDB3 base URL
+    And I am using an UiTID v1 API key of consumer "uitdatabank"
+    And I send and accept "application/json"
+    
+  Scenario: Suggest the ownership on an organizer of a created place
+    And I am authorized as JWT provider v1 user "centraal_beheerder"
+    And I create a minimal organizer and save the "id" as "organizerId"
+    And I keep the value of the JSON response at "url" as "organizerUrl"
+    And I am authorized as JWT provider v2 user "invoerder"
+    And I create a place from "places/place-with-organizer.json" and save the "id" as "placeId"
+    And I wait for the place with url "places/%{placeId}" to be indexed
+    When I send a GET request to "ownerships/suggestions/?itemType=organizer"
+    Then the response status should be 200
+    And the JSON response should include:
```

Comment:
Wouldn't it be better to be a bit more strict in how the response should look?

Reply:
That was something I first tried. But when running the acceptance tests locally for this user, the list of suggestions only grows because of previous runs. So we can only check if the new place or event is part of the query.